cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -504,6 +508,62 @@ export function changePropagateFrames(frames: number): AnyAction {
     };
 }
 
+
+//To remove annotation objects present in given range of frames
+export function removeAnnotationsinRangeAsync(sessionInstance: any, startFrame: number, endFrame: number, force: boolean): ThunkAction{
+    return async (dispatch: ActionCreator<Dispatch>): Promise<void> => {
+        const { filters, showAllInterpolationTracks } = receiveAnnotationsParameters();
+        try {
+
+            for(let frame = startFrame; frame<=endFrame; frame++){
+                await dispatch(changeFrameAsync(frame));
+
+                const states = await sessionInstance.annotations.get(frame, showAllInterpolationTracks, filters );
+
+                states.forEach(async (state: any) => {
+                    await   dispatch(removeObjectAsync(sessionInstance,state,force));
+                });
+            }
+
+            dispatch({
+                type: AnnotationActionTypes.REMOVE_ANNOTATIONS_INRANGE_SUCCESS,
+                payload: {
+                    startFrame,
+                    endFrame,
```

Comment:
Do you really need this payload? It was not changed in the function, so, I believe we do not need to pass it to reducers